src/features/landing/components/Header/AuthenticationButtons/AuthenticationButtons.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+
+import { Stack } from '@mui/material';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import CustomButton from '@/components/ui/UIButton/UIButton';
+
+function Buttons() {
```

Comment:
why name of the component is buttons? file component is AuthenticationButtons.tsx